docs/etl.md

```diff
@@ -101,7 +101,7 @@ $ ais put text.txt ais://src
 $ curl -s https://raw.githubusercontent.com/NVIDIA/ais-etl/master/transformers/md5/pod.yaml -o md5_spec.yaml
```

Comment:
in this part, we download the code, but the code has `    communication_type: ${COMMUNICATION_TYPE:-"\"hpull://\""}` we need to do a var substitution. It will still work without that, but it is recommended